graphql/rate-limit.ts

```diff
@@ -34,7 +35,7 @@ export function RateLimit(name: string, max: number, interval: number /* in ms *
 
         if (count > max) {
             log.warn(`Blocked ${context.ip} from accessing ${name} as maximum of ${max} was reached`);
-            throw new Error(`RateLimit Enforcement`);
+            throw new ForbiddenError(`RateLimit Enforcement`);
```

Comment:
Btw. I always wanted to ask why we have a rate limiter. Have we had any problems with too many requests in the past?

Reply:
@dhenkel92 no, mainly to prevent email / password enumeration, spam on the registration interface (hasn't happened yet, but better safe than sorry). Also it is abused in case the user app gets stuck in a retry loop and starts sending hundreds of backend requests. We also only Rate Limit a few specific endpoints for this (as rejecting unauthenticated users on authenticated queries is fast enough)